classes/assets/AbstractAssetManager.php

```diff
@@ -28,6 +28,8 @@
 
 abstract class AbstractAssetManagerCore
 {
+    use PrestaShop\PrestaShop\Adapter\Assets\AssetUrlGeneratorTrait;
```

Comment:
IMO these unrelated changes should be best left out of the PR

Reply:
I updated php-cs-fixer that's why these changes are here :(